pages/contact.vue

```diff
@@ -1,15 +1,109 @@
 <template>
-  <div class="white-wrapper">
-    Contact Page
+  <div class="background-section">
+    <div class="title">
+      <h1 class="cursive">Contact & Directions</h1>
```

Comment:
We should also make sure to add in "Pickup Only" or something like that. They don't do deliveries and that's something we wouldn't want to confuse the user with.

Reply:
Added Cash Only and No Delivery to the call section.